ManatiCR/manatime
# 39: Generate theme.

**Steps to test**
- Check if the new module manatime_theme_activator was added to the settings.php file.
- Verify the theme is activated automatically.
- Check if the ruby files were deleted.
- Review the manatime_theme_activator code, was created fine.

[Issue link](https://github.com/ManatiCR/manatime/issues/39)
